coffee-shop/style.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+* {
+    margin: 0px ;
+    padding: 0px ;
+}
+body {
+    background-color: blanchedalmond;
+}
+nav {
+   height: 100px;
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   flex-direction: column;
+   color: rgb(254, 90, 61);
+}
+nav > h1 {
+    text-transform: uppercase;
+    font-size: 50px;
+}
+.background-1 {
+    background-color: rgb(254, 90, 61);
+    padding: 20px 200px;
+}
+.header-img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 20px 0;
+}
+.coffee-img {
+    height: 500px;
+    margin: 10px 0;
+}
+.background-2 {
+    background-color: rgb(255, 198, 168);
+    padding: 20px 200px;
+}
+.menu {
+display: flex;
+justify-content: center;
+flex-wrap:wrap;
+}
+.menu > p {
+    font-size: 90px;
+    text-orientation: upright;
+    writing-mode: vertical-rl;
+    text-transform: uppercase;
+    position: absolute;
+    left: 10px;
+    top: 9em;
+}
+.box{
+    width: 250px;
+    height: 250px;
+    border: 3px solid rgb(252, 237, 189);
+    text-align: center;
+    margin: 10px;
+    padding: 10px;
+    cursor: pointer;
+}
+.no1 {
+    background-image: url(https://images.unsplash.com/photo-1514432324607-a09d9b4aefdd?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvZmZlZXxlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60);
+    background-position: center;
```

Comment:
Would be good to use class compositioning here,

```
.cardPosition: {
    background-position: center; 
    background-size: 320px;
}

cardBg1 {
   background-image: ...
}

cardBg2 {
   background-image: ...
}
```

Reply:
I tried your change to use class composition (the codes cleaner that way), but I didn't get it to work (the card's become a column). Maybe background-position: needs to be directly under background-image??

Reply:
@Dalitej oh yeah, maybe you're right, sorry, you could ignore my comment)